src/test/java/teammates/test/cases/datatransfer/FeedbackSessionAttributesTest.java

```diff
@@ -72,7 +72,8 @@ public void testBuilderCopy() {
                 .withEndTime(TimeHelperExtension.getInstantHoursOffsetFromNow(5))
                 .withSessionVisibleFromTime(TimeHelperExtension.getInstantHoursOffsetFromNow(1))
                 .withResultsVisibleFromTime(TimeHelperExtension.getInstantHoursOffsetFromNow(6))
-                .withTimeZone(8).withGracePeriod(0).withFeedbackSessionType(FeedbackSessionType.PRIVATE)
+                .withTimeZone(TimeHelper.convertToZoneId(8)).withGracePeriod(0)
```

Comment:
Just use something like `ZoneId.of("+08:00")`? `convertToZoneId` should be removed eventually.